### PR TITLE
chore(docs): added slack link for one-amex

### PR DESCRIPTION
**Pull request by shivaylamba**
Added link for Slack for One-Amex #129 

## Description
Added link for Slack for One-Amex in Contributing.md

## Motivation and Context
Adds Slack group link for One-Amex

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation (adding or updating documentation)

## Checklist:

- [X] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

## What is the Impact to Developers Using One App?
Helps them join the community on Slack.

### CODE REVIEW

**Comment by nellyk** [comment]
Thank you for your contribution. This looks good!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -102,6 +102,10 @@ To auto fix eslint rule failures that are autofixable run `npm run test:lint --
 
 * Write tests for your changes. A feature is not considered done until it has tests and/or a test plan. It does not matter if code coverage shows 100%, tests are expected for *all* changes. This includes [integration tests](./__tests__/integration/README.md) as needed.
 
+### Getting in contact
```
Comment (Francois-Esquire) [inline CONTRIBUTING.md:105]:
```suggestion
### Getting In Contact
```
To keep consistent with the rest of the document, could we capitalize this please?

**Comment by shivaylamba** [comment]
@Francois-Esquire Changed the commit

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -102,6 +102,10 @@ To auto fix eslint rule failures that are autofixable run `npm run test:lint --
 
 * Write tests for your changes. A feature is not considered done until it has tests and/or a test plan. It does not matter if code coverage shows 100%, tests are expected for *all* changes. This includes [integration tests](./__tests__/integration/README.md) as needed.
 
+### Getting in Contact
+
+- Join our [Slack channel](https://one-amex.slack.com) request an invite [here](https://join.slack.com/t/one-amex/shared_invite/enQtOTA0MzEzODExODEwLTlmYzI1Y2U2ZDEwNWJjOTAxYTlmZTYzMjUyNzQyZTdmMWIwZGJmZDM2MDZmYzVjMDk5OWU4OGIwNjJjZWRhMjY)
```
Comment (jsmartfo) [inline CONTRIBUTING.md:107]: please link "invite" not "here"

- Join our [Slack channel](https://one-amex.slack.com) request an [invite](https://join.slack.com/t/one-amex/shared_invite/enQtOTA0MzEzODExODEwLTlmYzI1Y2U2ZDEwNWJjOTAxYTlmZTYzMjUyNzQyZTdmMWIwZGJmZDM2MDZmYzVjMDk5OWU4OGIwNjJjZWRhMjY)

Comment (shivaylamba) [reply CONTRIBUTING.md:107]: @jsmartfo Where shall the invite be shown

Comment (Francois-Esquire) [reply CONTRIBUTING.md:107]:
```suggestion
- Join our [Slack channel](https://one-amex.slack.com) request an [invite](https://join.slack.com/t/one-amex/shared_invite/enQtOTA0MzEzODExODEwLTlmYzI1Y2U2ZDEwNWJjOTAxYTlmZTYzMjUyNzQyZTdmMWIwZGJmZDM2MDZmYzVjMDk5OWU4OGIwNjJjZWRhMjY)
```

**Comment by shivaylamba** [comment]
Is this good to be merged? @jsmartfo

**Pull request merged**